eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -70,7 +70,7 @@ jobs:
   - template: /eng/common/templates/variables/pool-providers.yml
   - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
     - group: AzureDevOps-Artifact-Feeds-Pats
-  - ${{ if and(not(parameters.isBuiltFromVmr), eq(variables['System.TeamProject'], 'internal'), not(startswith(parameters.vmrBranch, 'internal/release/'))) }}:
+  - ${{ if and(and(not(parameters.isBuiltFromVmr), eq(variables['System.TeamProject'], 'internal'), not(startswith(parameters.vmrBranch, 'internal/release/'))), not(eq(variables['Build.Reason'], 'PullRequest'))) }}:
```

Comment:
You shouldn't need nested `and` functions. The `and` function allows N arguments that are all `and`ed  together. Keeps it a bit more readable.